src/Sdk/DTLogging/Logging/ValueEncoders.cs

```diff
@@ -118,7 +118,8 @@ private static String TrimBase64End(String value)
             if (value.EndsWith('='))
             {
                 var trimmed = value.TrimEnd('=');
-                if (trimmed.Length > 1)
+                // Minimum length to make sure we don't register 1 or 2 characters as a secret.
+                if (trimmed.Length > 3)
```

Comment:
For original strings (not base 64 encoded versions) < 4 characters we will not trim (like we did in the past before the recent changes in #284 .
This includes strings that are 4 or 5 characters, but when shifted via the base 64 encoders then end up being < 4 characters. The 4 or 5 character encoders will trim, but the shifted versions that end up being length < 4 will not.